### ISSUE TITLE
[Notification] ENVO:'Satellite imaging'

### ISSUE THREAD

**Issue by pbuttigieg** [BiodiversityOntologies/bco#90]
Not exactly "biological" but part of the whole observations discussion, thus this is more an FYI. Feel free to link to other issues in, e.g., OBI. 

We have a request for a remote sensing class here: https://github.com/EnvironmentOntology/envo/issues/629

We can, of course, create a class for this as requested. ENVO does offer the semantics for the stuff being sensed and the import is unidirectional. We may branch off an "environmental observation" ontology in the same family as BCO. That being said, we'd like to plug this in to a higher-level ontology for "observations" in general.

@cmungall tagged for thoughts on modularisation.

### xrefs:
https://github.com/BiodiversityOntologies/bco/issues/86
https://github.com/BiodiversityOntologies/bco/issues/85

**Comment by pbuttigieg** [comment]
PS: The ENVO request asks for a synonym "remote sensing" which is actually quite a bit broader than satellite imagery. That's a great example of what should go in a dedicated observation ontology.

**Comment by ramonawalls** [comment]
> PS: The ENVO request asks for a synonym "remote sensing" which is actually quite a bit broader than satellite imagery. That's a great example of what should go in a dedicated observation ontology.

And a great reason for someday getting back to the observation ontology alignment work.

**Comment by dr-shorthair** [comment]
'remote sensing' was one of the motivations for the structure used in OGC O&M which became ISO 19156 and later W3C SSN. 
Separation of the act-of-observation from the feature-of-interest and the sensor allows them to be in different places if necessary. This allows all combos of in-situ, ex-situ and remote sensing scenarios to use a common language.

**Comment by robgur** [comment]
This conversation is very relevant to a workshop I attended last week at
Cal Tech sponsored by the Keck Institute of Space Sciences and related to
next steps for remote sensing biodiversity.   We really need to do both
data integration via the route we are discussing here and via modeling to
create assimilated/integrated products.  There is going to be a lot of
support for just these sorts of activities.

**Comment by pbuttigieg** [comment]
@dr-shorthair 
>Separation of the act-of-observation from the feature-of-interest and the sensor allows them to be in different places if necessary. 

Yes, it was a wise design decision. This slots in nicely with the processes being in an obs ontology and the features of interest being in other domain ontologyes (ENVO for planetary science, UBERON for anatomy, CHEBI for chemicals, etc) 

Thanks @robgur - I think we should look for some support for this work, having some students on this can speed it up quite a bit.
